src/components/tabs/js/tabsController.js

```diff
@@ -558,17 +562,22 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
   }
 
   /**
+   * Calculates the width of the pagination wrapper by summing the widths of the dummy tabs.
    * @returns {number}
    */
   function calcPagingWidth () {
-    var width = 1;
+    return calcTabsWidth(getElements().dummies);
+  }
+
+  function calcTabsWidth(tabs) {
+    var width = 0;
```

Comment:
I'll also note that this is added in response to issues with tabs living inside of dialogs, so you might try creating that sort of test case and see if any bugs show up.